setup.py

```diff
@@ -98,6 +95,7 @@
     'pytest-cache>=1.0',
     'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
+    'pytest-runner>=2.7.0',
```

Comment:
Ah, found the bug. `pytest-runner` should be in the `setup_require` section, not the `tests_require`.